src/components/forms/radio-button-group/RadioGroup.tsx

```diff
@@ -38,38 +37,39 @@ const rootStyles = {
       },
     }),
 };
-export function RadioGroup(props: RadioGroupProps) {
+export function RadioButtonGroup(props: RadioButtonGroupProps) {
```

Comment:
My suggestion for this component was that you make the API identical to blueprint's `RadioGroup` component. You would spread the props as is from `RadioGroupButton` to blueprint's `RadioGroup`